build_container/build_container_centos.sh

```diff
@@ -1,6 +1,9 @@
 #!/bin/bash
 
 set -e
+
+export PATH="/opt/rh/rh-git218/root/usr/bin:/opt/rh/devtoolset-7/root/usr/bin:/opt/llvm/bin:/usr/local/sbin:/usr/local/bin:${PATH}"
```

Comment:
I can push the image if you want

Reply:
We had a similar issue on a build server, with RVM. The solution, because jenkins runs as a service, and not a login shell, was to add the call to ". /etc/profile.d/rvm.sh" ("." is an alias of "source") to the user's ".bash_profile". A similar fix might work here.